src/components/SQForm/SQFormRadioButtonGroup.tsx

```diff
@@ -47,7 +48,15 @@ function SQFormRadioButtonGroup({
     fieldState: {isFieldError, isFieldRequired},
     formikField: {field},
     fieldHelpers: {handleChange, handleBlur, HelperTextComponent},
-  } = useForm({name, onChange});
+  } = useForm<
+    RadioButtonInputItemProps['value'],
+    React.ChangeEvent<HTMLInputElement>
+  >({
+    name,
+    onChange,
+  });
+
+  console.log('field', field);
```

Comment:
Stray console.log 👀

Reply:
Should we add a rule to eslint to catch these for us? I know I'm not the only one that does this lol

Reply:
Yeah I imagine there's a rule for it. As long as it isn't a pain/blocker while we're actively working on something

Reply:
Ya. Our dev environment is super stingy with eslint errors and blocks the build/compile when it seems them. That may be why we don't have that rule in place. I wonder how hard it'd be to setup using a different set of rules in our CI and precommit `npm run lint` checks to catch things like this but them through while actively developing and running `npm run storybook`

Reply:
Setting console logs to warn instead of error will prevent the dev environment (storybook) from failing the compilation.
To my knowledge, the `npm run lint` script will only fail on errors, not warnings.

```
// eslint config
  "rules": {
    "no-console": "warn",
    "no-debugger":"warn"
  }
  ```